src/components/button/button.css

```diff
@@ -2,12 +2,8 @@
 .cx-btn__secondary,
 .cx-btn__tertiary,
 .cx-btn__danger {
-  --highlight-color: var(--cx-color-blue);
-  --highlight-background: var(--cx-color-grey-700);
-  --highlight-size: 500%;
-  --background-transition-duration: 150ms;
+  --cx-gradient-highlight-size: 500%;
```

Comment:
Jeg har skrevet litt om button-komponenten. I stedenfor å bruke et `::before` element for å lage gradienten, så animerer vi nå custom properties i stedet. Da slipper vi `::before` elementet, og koden er hakket enklere å lese! Da kan også animasjonen gjenbrukes i chip-komponenten.

Reply:
😎